clang/lib/AST/ItaniumMangle.cpp

```diff
@@ -6803,9 +6809,6 @@ bool CXXNameMangler::mangleSubstitution(QualType T) {
 }
 
 bool CXXNameMangler::mangleSubstitution(TemplateName Template) {
-  if (TemplateDecl *TD = Template.getAsTemplateDecl())
-    return mangleSubstitution(TD);
-
```

Comment:
Can you explain that change?

Reply:
This one looks like it was a rebasing mistake. Restored.